SPARQLWrapper/Wrapper.py

```diff
@@ -349,6 +351,18 @@ def setOnlyConneg(self, onlyConneg):
         """
         self.onlyConneg = onlyConneg
 
+    def setCustomHeaders(self, customHeaders):
```

Comment:
why not also a `addCustomHeader(self, header, value)` method?

Reply:
I like the idea. I am going to change the code in order to take this approach!